### PR TITLE
Adds support for CDK v1

### PR DESCRIPTION
**Pull request by nroldanf**
## Description

- Adds support for CDK v1
- Adds AWS Chalice based API to submit jobs

Notes:
- [x] I have tested infrastructure end-to-end.

### CODE REVIEW

File: infrastructure/cdkv1/lokafold-app/.chalice/deployed/dev.json [hunk 1]
```diff
@@ -0,0 +1,23 @@
+{
```
Comment (jjpulidos) [inline infrastructure/cdkv1/lokafold-app/.chalice/deployed/dev.json:1]: Do we need this file?

**Review by jpvanegasc — APPROVED** [review]
LGTM

File: infrastructure/cdkv1/lokafold-app/app.py [hunk 2]
```diff
@@ -0,0 +1,115 @@
...
+
+logging.basicConfig(
+    format="%(asctime)s %(levelname)-s:%(name)s: %(message)s",
+    level=logging.INFO,
+    datefmt="%H:%M:%S",
+    stream=sys.stderr
+)
+logger = logging.getLogger(__name__)
+
+@app.route("/", methods=["POST", "GET"])
```
Comment (jjpulidos) [inline infrastructure/cdkv1/lokafold-app/app.py:29]: So as we are not using DELETE, PATCH and PUT, can we erase those from the policies role?

File: infrastructure/cdkv1/lokafold-app/app.py [hunk 3]
```diff
@@ -0,0 +1,115 @@
...
+        except Exception as err:
+            raise f"Error submiting the jobs {err}"
+        
+        response = {
+            "feature_extraction_job_response": step_1_response,
+            "prediction_job_response": step_2_response,
+        }
+        return response
+    else:
+        return {"Error": "Method is not implemented. Use either GET or POST"}
```
Comment (jjpulidos) [inline infrastructure/cdkv1/lokafold-app/app.py]: Can we add and enpty endline for all the files?

File: infrastructure/cdkv1/lokafold-app/app.py (hunk 2 above)
Comment (nroldanf) [reply infrastructure/cdkv1/lokafold-app/app.py:29]: No... those errors arise even when those methods weren't implemented

**Review by jjpulidos — APPROVED** [review]
Nice work done here!

**Pull request merged**